app/features/industry/registration.feature

```diff
@@ -0,0 +1,12 @@
+@mvp
+Feature: Registration
+
+  As an industry reporter
+  I want to be able to onboard myself to the CIIP application portal
+  So that I can access business confidential information
+
+  Scenario: invalid session redirects to login page
+  Scenario: import user data from valid session after login
+  Scenario: re-import user data from valid session after login when session attributes have changed
+  Scenario: registration page allows the user to edit their details
+  Scenario: accept the legal disclaimer
```

Comment:
@tenlig please make sure we schedule the feature for MVP -- talked to @wenzowski  and it is in scope.